### DISCUSSION TITLE
`abortOnError` option

### DISCUSSION THREAD

**Discussion by Maeda-Naoki** [biomejs/biome#2466] [Ideas]
In order to display Lint results in CI results, I would like to have an `abortOnError` option that would result in an `exit code 0` even if a warning exists.
Target commands  `check`, `lint`, `ci`.

Ref: [Andrlid Lint : abortOnError](https://developer.android.com/reference/tools/gradle-api/7.3/com/android/build/api/dsl/Lint#abortOnError())

**Comment by ematipico** [comment]
I think `--error-on-warnings` does what you asked https://biomejs.dev/reference/cli/#biome-version

**Reply by Maeda-Naoki** [reply]
@ematipico   
Thank you for your response.
What I want to do is to exit the command with `exit code 0` even if there are warnings.
The `--error-on-warnings` looks like an option to end with `exit code 1` when warnings are present.  
What I'm trying to say is whether it is possible to have `abortOnError = false` as in Android Lint.

**Reply by ematipico** [reply]
Have you tried `--ignore-errors`?

**Reply by Maeda-Naoki** [reply]
Thank you for your response.
However, the problem has not been solved.

* When I run with --ignore-errors, I get the following error.
  * `Error: --ignore-errors is not expected in this context `
* There is no description of `--ignore-errors` in https://biomejs.dev/reference/cli/.

@ematipico 
What version are you using?

**Reply by ematipico** [reply]
Thank you @Maeda-Naoki 

There's an option called `--skip-errors`, but it seems it isn't wired in our CLI.

We should really make sure to implement such a feature. Thank you for your patience!
